src/components/App.js

```diff
@@ -1,15 +1,67 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../logic/calculate';
 
-calculate(1, 2, '-');
+class App extends Component {
+  constructor(props) {
+    super(props);
 
-const App = () => (
-  <>
-    <Display />
-    <ButtonPanel />
-  </>
-);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+      lastClicked: null,
+    };
+  }
+
+  handleClick = buttonName => {
+    const {
+      total, next, operation, lastClicked,
+    } = this.state;
+    const operations = ['+', '-', '÷', 'X', '='];
+    console.log(lastClicked);
+    if (buttonName === '.') {
+      this.setState({
```

Comment:
let move this code calculate.js, handleclick would then rely on the logic in the calculate.js.